packages/components/src/Dropzone/sub-components/SelectedFileMessage/SelectedFileMessage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { Button } from "@kaizen/button"
+import { Icon } from "@kaizen/component-library"
+import selectedFileSvgIcon from "@kaizen/component-library/icons/file.icon.svg"
+import acceptedFileSvgIcon from "@kaizen/component-library/icons/success.icon.svg"
+import { Paragraph } from "@kaizen/typography"
+import { trunicateFilename } from "../../utils"
+import { Hyperlink } from "../Hyperlink/Hyperlink"
+
+interface SelectedFileMessageProps {
+  fileName: string
+  handleOpenDialog: () => void
+}
+
+export const AcceptedFileIcon = (): JSX.Element => (
+  <span className="inline-block mb-8 text-green-500">
+    <Icon
+      height={20}
+      width={20}
+      icon={acceptedFileSvgIcon}
+      title="Selected File"
+      role="presentation"
+      inheritSize={true}
```

Comment:
Wrapping this icon in `inline-block` isn't necessary if you remove the `inheritSize`